### ISSUE TITLE
Minor formatting related suggestions

### ISSUE THREAD

**Issue by lmanan** [dlmbl/image_segmentation#7]
Hi Will. Great exercise. I had some nitpicking suggestions.

Formatting related: 
1. Maybe we can consistently refer to U-Net and PyTorch. (sometimes, U-Net is without a hyphen, PyTorch is referred to as Pytorch etc) 
2. Spot the best U-Net is a lovely thought-provoking question. But don't we already provide the solution in the cell afterwards? Maybe that solution cell should go
3. Small typo in the spelling of Dice Coefficient in `We will use the [Dice Coefficeint] to evaluate the network predictions.`
4. Small typo in the word "addition" here `In adddition, it has skip connections, that bridge the output from an encoder to the corresponding decoder`.
5. In colored box 5.2, you could use `<a href>` to make a link around `Pytorch tutorial` instead of markdown link.
6. Maybe as an optional thing, would be good to add a comment about  what `0.5, 0.5` means in :
```
self.inp_transforms = transforms.Compose(
            [
                transforms.Grayscale(),  # some of the images are RGB
                transforms.ToTensor(),
                transforms.Normalize([0.5], [0.5]),
            ]
        )
```
7. Typo in `Task BONUS.3: Dice Loss Dice loss is a simple inversion of the dice coefficient. We already have a dice coefficient implementation, so now we just need a layer that can invert it.` (`Dice Loss` repeated twice ...)
8. Section 4 called as `Loss and distance metrics` could be referred to as `Loss function` since there is no subsequent reference to the notion of distance in the notebook.

**Comment by pattonw** [comment]
Perfect! I agree with all these changes and looks like you merged most of them already. I didn't see a comment about the meaning of 0.5 in the normalize transform, but that would be good to add. I'll go ahead with that if its not done yet
